### ISSUE TITLE
Rendering numbers and strings

### ISSUE THREAD

**Issue by genmeblog** [nextjournal/clerk#114]
Two minor issues.

* floating point number with fractional part = 0 is rendered as an integer
* strings are double quoted (ok) but in tables are not

```
[1.0 2.0 3.0 4.5 5.0]
;; [1 2 3 4.5 5]
;; should be [1.0 2.0 3.0 4.5 5.0]

(map str [1.0 2.0 3.0 4.5 5.0])
;; ("1.0" "2.0" "3.0" "4.5" "5.0")

(clerk/table
 [[[1.0 2.0 3.0 4.5 5.0]]
  [(map str [1.0 2.0 3.0 4.5 5.0])]])
;; [1 2 3 4.5 5]
;; (1.0 2.0 3.0 4.5 5.0) ;; should be ("1.0" "2.0" "3.0" "4.5" "5.0")
```

**Comment by mk** [comment]
Thanks for the report. The number is probably due to javascript representation of numbers and something we'll look into fixing.

That strings aren't quoted in tables is intentional as it's how spreadsheet software will display strings. You can change it by removing the viewer from `!viewers`:
https://github.com/nextjournal/clerk/blob/8a44eba2a2fd99e6e9350cfafddf6bdea1c07bd3/src/nextjournal/clerk/viewer.cljc#L243

**Comment by genmeblog** [comment]
Ok, thanks.

**Comment by mk** [comment]
The number display has been reported again in #335 and fixed with 766400ea75a807a95926a250bd3186eefef45804.

**Comment by genmeblog** [comment]
Btw. rendering floats without fractional part as ints is still the case. 
![image](https://user-images.githubusercontent.com/38646601/222535315-b1d54dc4-fa82-4903-9210-bd67d3873031.png)
